ledger-core/CMakeLists.txt

```diff
@@ -31,6 +31,10 @@ set(CMAKE_MACOSX_RPATH 1)
 
 add_definitions("-DSQLITE_HAS_CODEC")
 
+set(INSTALL_TARGETS
+
+)
+
```

Comment:
What's the purpose of this variable ? Does it get filled automatically when calling `install` later ?

Reply:
Good catch ! I will remove it :trollface: